ep008/rev08/t.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python2
```

Comment:
Added this in order to signify that it should be executed with python version 2.